### DISCUSSION TITLE
Initial Sync

### DISCUSSION THREAD

**Discussion by jrich523** [savvasdalkitsis/uhuruphotos-android#1133] [Q&A]
How do i tell it to sync my entire photo library? When i run a "full feed sync" it just grabs whats new. I've set the window to the 30 day limit and it doesnt even sync the 30 day window. 

When viewing I am able to see all the images from my camera, but from a sync standpoint it only does new stuff. 

Pixel 7 Pro

Thanks

**Comment by savvasdalkitsis** [comment]
Hello. So the feed sync changed in the latest release but I haven't pushed it to the play store yet cause you need to be running the latest dev version of librephotos for it to work.

In the new version, the first time you sync, it will sync everything from the server and then every sync will only do incremental fetches. 

The current version provides the full sync job as a way for the client to fetch changes that MAY have happened on the server in older days as there was no way for the client to be told when new changes happened to old media (let's say you deleted a photo from a year ago).

The latest dev version of librephotos has a way for the client to get only new changes (additions, deletions etc) since the last time it checked so going forward this will be the standard way of synchronising with the server.

If you are running the latest dev of librephotos, you can grab the latest uhuruphotos release from the github page if you don't want to wait until i publish it to the play store

**Reply by jrich523** [reply]
thanks! i'll give that a try, im not scared to try some dev builds :)

**Comment by jrich523** [comment]
Hi @savvasdalkitsis,

I've updated to 2024w40 and 0.0.234 and im still experiencing the same thing. (only sync's the first couple of days)

Really the goal is to backup all the photos from my phone to my server (and backup the server to the cloud), and be able to remove some from my phone.
The root of the problem is my phone is out of storage and I need to delete some images to free up space. Ideally it would be great if it synced all and just auto removed some phone photos based on age or something.

Thanks! appreciate your effort on this project!
